sdcm/tester.py

```diff
@@ -354,6 +354,9 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
 
     @cached_property
     def argus_test_run(self):
+        if self.params["disable_argus"]:
+            self.log.info("Argus is disabled by configuration")
```

Comment:
The same here, need to add an SCT event of the "WARNING" type.
We should have an event for such significant thing as disabling argus.

Reply:
So I guess this should be the case for lines 366 and 367 further down? :slightly_smiling_face: 
If yes, I will add a different PR to make this into an event also.

Reply:
Right